.github/workflows/build-test-release.yml

```diff
@@ -325,8 +325,7 @@ jobs:
     needs: build-test-addon
     if: |
       !cancelled() &&
-      needs.build-test-addon.result == 'success' &&
-      ( github.base_ref == 'main' || github.ref_name == 'main' )
+      needs.build-test-addon.result == 'success'
```

Comment:
not sure what was limitation to run it only for main but we can go with main or develop also